.github/ISSUE_TEMPLATE/command_request.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
duplicate of the above. why have both? as a user how would I know which one to pick?

Reply:
If you are requesting a new feature to be added, verses going to be adding a new feature yourself and what to discuss it with us beforehand.

I think im going to remove command_request tho